src/vivarium_gates_nutrition_optimization/constants/results.py

```diff
@@ -29,14 +29,14 @@
 DEATH_COLUMN_TEMPLATE = "death_due_to_{CAUSE_OF_DEATH}_age_{AGE_GROUP}"
 YLLS_COLUMN_TEMPLATE = "ylls_due_to_{CAUSE_OF_DEATH}_age_{AGE_GROUP}"
 YLDS_COLUMN_TEMPLATE = "ylds_due_to_{CAUSE_OF_DISABILITY}_age_{AGE_GROUP}"
-STATE_PERSON_TIME_COLUMN_TEMPLATE = "{STATE}_person_time_age_{AGE_GROUP}"
+PREGNANCY_STATE_PERSON_TIME_COLUMN_TEMPLATE = "pregnancy_{PREGNANCY_STATE}_person_time_age_{AGE_GROUP}"
```

Comment:
nit: the "pregnancy_" part may be redundant b/c they'll look like "pregnancy_pregnant_person_time..." and "pregnancy_not_pregnant_person_time...."

Reply:
Is the concern cosmetic? IMHO, these are intermediate results (RT never touches these output.hdf file) and it's not unreasonable to have a prefix like this when/if you add other person time observers.

From an efficiency point of view, we tend to bias toward readability over efficient usage of bytes, for better or worse.